tests/RequestCookiesTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\RequestProvider\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\ServerRequestInterface;
+use Yiisoft\Di\Container;
+use Yiisoft\Di\ContainerConfig;
+use Yiisoft\RequestProvider\RequestCookies;
+use Yiisoft\RequestProvider\RequestProviderInterface;
+
+use function dirname;
+
+final class RequestCookiesTest extends TestCase
+{
+    public function testRequestCookiesGet(): void
+    {
+        $requestCoockies = $this->getRequestCookies();
+
+        $this->assertSame('value', $requestCoockies->get('test'));
+    }
+
+    public function testRequestCookiesHas(): void
+    {
+        $requestCoockies = $this->getRequestCookies();
+
+        $this->assertTrue($requestCoockies->has('test'));
+
+        $this->assertFalse($requestCoockies->has('value'));
+    }
+
+    private function getRequestCookies(): RequestCookies
+    {
+        $container = $this->createContainer();
```

Comment:
Here we test `RequestCookies` only, so we can don't use container and create `RequestProvider` directly.